projects/plugins/jetpack/extensions/shared/currencies.js

```diff
@@ -41,7 +46,7 @@ export const SUPPORTED_CURRENCIES = {
  */
 export const CURRENCY_OPTIONS = Object.keys( SUPPORTED_CURRENCIES ).map( value => {
 	const { symbol } = getCurrencyDefaults( value );
-	const label = symbol === value ? value : `${ value } ${ trimEnd( symbol, '.' ) }`;
+	const label = symbol === value ? value : `${ value } ${ removeTrailingDots( symbol ) }`;
```

Comment:
https://caniuse.com/?search=trimEnd shows good browser support these days.
```suggestion
	const label = symbol === value ? value : `${ value } ${ String( symbol || '' ).trimEnd( '.' ) }`;
```
or [if `symbol` is always a string already](https://github.com/Automattic/wp-calypso/blob/%40automattic/format-currency%401.0.1/packages/format-currency/src/types.ts#L17),
```suggestion
	const label = symbol === value ? value : `${ value } ${ symbol.trimEnd( '.' ) }`;
```

Reply:
Unfortunately, it looks like the native version of `trimEnd` doesn't take any parameters, which means that we can't use it for the purpose of removing trailing dots 😕

Reply:
Crap, I should have looked closer. Sorry for the bad recommendation.